rosetta-testing/src/test/java/com/regnosys/rosetta/generator/java/object/ConfigurableTypeTabulatorTest.xtend

```diff
@@ -1,15 +1,71 @@
 package com.regnosys.rosetta.generator.java.object
 
-import org.junit.jupiter.api.Test
 import com.regnosys.rosetta.config.file.RosettaConfigurationFileProvider
-
-import static extension com.regnosys.rosetta.tests.util.CustomConfigTestHelper.*
 import java.net.URL
-import static org.junit.jupiter.api.Assertions.*
-import static org.hamcrest.MatcherAssert.*
 import org.hamcrest.CoreMatchers
+import org.junit.jupiter.api.Test
+
+import static org.hamcrest.MatcherAssert.*
+import static org.junit.jupiter.api.Assertions.*
+
+import static extension com.regnosys.rosetta.tests.util.CustomConfigTestHelper.*
 
 class ConfigurableTypeTabulatorTest {
+
+	@Test
+	def void shouldGenerateTabulatorsForSingleTypeListedInConfig() {
```

Comment:
what is the significance of this test?

Reply:
Testing tabulation of the an the simplest possible type using the default testing namespace com.rosetta.test.model which was previously not working